.gitignore

```diff
@@ -11,3 +11,5 @@ secrets/
 *.tfstate
 *.tfstate.backup
 
+# Allow additional Terraform configuration, f.ex. to create tags
+/custom.tf
```

Comment:
I think being able to customize tags can come afterwards. We should decide on tag names for etcd, workers and masters and set those by default :)